crates/bcr-wallet-core/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "bcr-wallet-core"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+tokio_with_wasm = "0.8.5"
+wasm-bindgen = "0.2"
+js-sys = "0.3.77"
+wasm-bindgen-futures = "0.4.50"
+console_error_panic_hook = "0.1.7"
+gloo-timers = { version = "0.3.0", features = ["futures"] }
+tsify = { version = "0.4.5", features = ["js"] }
+thiserror.workspace = true
+serde.workspace = true
+async-trait.workspace = true
+log = "0.4.27"
+reqwest = { workspace = true, features = ["json"] }
+anyhow.workspace = true
+rand.workspace = true
+tracing.workspace = true
+tracing-subscriber.workspace = true
+tracing-wasm = "0.2.1"
+getrandom = { version = "0.2.16", features = ["js"] }
```

Comment:
I guess you had a build error and added this because of it? I'm just asking because that's quite an old version